.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,7 @@ assignees: ""
 
 <!-- A clear and concise description of what the bug is. -->
 
-**To Reproduce** Steps to reproduce the behavior:
+**To Reproduce** Steps to reproduce the behaviour:
```

Comment:
nit: both are correct, depending on whether you choose UK or US english as baseline.

Reply:
Yeah, I just picked it at random to match the other occurrence in the same file.